api/transaction_args.go

```diff
@@ -74,6 +87,59 @@ func (arg *TransactionArgs) data() []byte {
 	return nil
 }
 
+// CallDefaults sanitizes the transaction arguments, often filling in zero values,
+// for the purpose of eth_call class of RPC methods.
+func (args *TransactionArgs) CallDefaults(globalGasCap uint64, baseFee *big.Int, chainID *big.Int) error {
```

Comment:
We already have a [setDefaults](https://github.com/openrelayxyz/cardinal-evm/blob/ac2349be446347f6a6e58d16ea179942c017271b/api/transaction_args.go#L148) method. There might be some things that need to be added to it, but this seems duplicative.